uk.co.compucorp.civicrm.hrcontactactionsmenu/js/contactactions.js

```diff
@@ -23,30 +22,21 @@
     });
   }
 
-  /**
-   * Initializes click listeners for
-   * - Delete User Account Button
-   * - Actions Button
-   */
-  function initListeners () {
-    confirmUserAccountDeletion();
-    toggleActionMenuButtonClass();
-  }
-
   /**
    * Toggles the "active" class in "Actions" button
    *
    * @param {Object} event
    */
-  function toggleActionMenuButtonClass (event) {
+  function markMenuButtonAsActiveWhenPressed (event) {
     $(document).on('click', function (event) {
-      var selector = $('#crm-contact-actions-link');
+      var $button = $('#crm-contact-actions-link');
+      var $target = $(event.target);
 
-      if ($(event.target).is('#crm-contact-actions-link, #crm-contact-actions-link *')) {
-        selector.addClass('active');
+      if ($target.is($button) || !!$button.has($target).length) {
```

Comment:
I don't think you need `!!` here. `!!` is helpful when you want to return a boolean in the function, but you have an integer, for example. In this case no-one cares if it is an integer or bool. Not sure it adds readability in case of `length` either.